plugins/reach/lib/api/data/entryVendorTaks/KalturaQuizVendorTaskData.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/**
+ * @package plugins.reach
+ * @subpackage api.objects
+ * @relatedService EntryVendorTaskService
+ */
+class KalturaQuizVendorTaskData extends KalturaVendorTaskData
+{
+	/**
+	 * Number Of Questions.
+	 *
+	 * @var int
+	 */
+	public $numberOfQuestion;
+
+	/**
+	 * Questions Type.
+	 *
+	 * @var string
+	 */
+	public $questionsType;
```

Comment:
is there a closed number of types for this field and it can be enum?

Reply:
this is for future use probably will be json so no enum or any type.